packages/gafl-webapp-service/src/pages/order-complete/order-complete/route.js

```diff
@@ -31,11 +31,12 @@ export const getData = async request => {
   await request.cache().helpers.status.set({ [COMPLETION_STATUS.completed]: true })
   await request.cache().helpers.status.setCurrentPermission({ currentPage: ORDER_COMPLETE.page })
   const startTimeStringTitle = displayStartTime(request, permission)
+  const permissionCost = getPermissionCost(permission)
```

Comment:
when getPermissionCost is called elsewhere it has parameters of startDate and permit, will this work with permission?

Reply:
Yes, because it's the permission that's in the transaction cache. The issue elsewhere is that we're making, essentially, an HTTP request with the permission by sending to the Sales API, and so we reduce the size of the payload by stripping out the permit details other than the permit id and then requesting this from the Sales API.